servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractResteasyTest.java

```diff
@@ -420,4 +438,31 @@ public void testGetContent() {
 
     assertThat(content).isEqualTo(table);
   }
+
+  @Test
+  public void testGetDiffs() {
+    Branch fromBranch = makeBranch("getdiffs-test-from");
+    Branch toBranch = makeBranch("getdiffs-test-to");
+    IcebergTable fromTable = IcebergTable.of("content-table", 42, 42, 42, 42);
+    IcebergTable toTable = IcebergTable.of("content-table", 43, 43, 43, 43);
+
+    ContentKey contentKey = ContentKey.of("key1");
+    commit(contentKey, fromTable, fromBranch, "diffAuthor");
+    commit(contentKey, toTable, toBranch, "diffAuthor2");
+
+    DiffResponse diffResponse =
+        rest()
+            .get(String.format("diffs/%s...%s", fromBranch.getName(), toBranch.getName()))
+            .then()
+            .statusCode(200)
+            .extract()
+            .as(DiffResponse.class);
+
+    assertThat(diffResponse).isNotNull();
+    assertThat(diffResponse.diffs()).hasSize(1);
+    Diff diff = diffResponse.diffs().get(0);
+    assertThat(diff.key()).isEqualTo(contentKey);
+    assertThat(diff.from()).isEqualTo(fromTable);
+    assertThat(diff.to()).isEqualTo(toTable);
```

Comment:
add test cases to add/delete differences?

Reply:
added that to `AbstractTestRest`